content/en/agent/faq/agent-v6-changes.md

```diff
@@ -0,0 +1,564 @@
+| `collector_log_file`        | `log_file`                   |                                                                                                   |
+| `syslog_host`               | `syslog_uri`                 | The Syslog configuration is now expressed as a URI.                                               |
+|                             | `syslog_pem`                 | Syslog configuration client certificate for TLS client validation.                                |
+|                             | `syslog_key`                 | Syslog configuration client private key for TLS client validation.                                |
+
+##### Removed
+
+| Name                         | Notes                                                                                                                 |
+|------------------------------|-----------------------------------------------------------------------------------------------------------------------|
+| `proxy_port`                 | Superseded by `proxy`, see the [proxy][3] documentation for details.                                                  |
+| `proxy_user`                 | Superseded by `proxy`, see the [proxy][3] documentation for details.                                                  |
+| `proxy_password`             | Superseded by `proxy`, see the [proxy][3] documentation for details.                                                  |
+| `proxy_forbid_method_switch` | Obsolete                                                                                                              |
+| `use_mount`                  | Deprecated at the Agent-level and moved to the [Disk check][4].                                                       |
+| `device_blacklist_re`        | Deprecated at the Agent-level and moved to the [Disk check][4] as `device_blacklist`.                                 |
+| `use_curl_http_client`       | Obsolete                                                                                                              |
+| `exclude_process_args`       | Deprecated feature                                                                                                    |
+| `check_timings`              | Superseded by internal stats                                                                                          |
+| `non_local_traffic`          | Superseded by `dogstatsd_non_local_traffic` for Dogstatsd and `apm_config.apm_non_local_traffic` for the Trace Agent. |
+| `dogstatsd_target`           |                                                                                                                       |
+| `dogstreams`                 | Deprecated feature, use the [Logs Agent][5] instead.                                                                  |
+| `custom_emitters`            |                                                                                                                       |
```

Comment:
What happened to this parameter ?

Reply:
It's in the removed list but there were no notes about it on current doc